modules/ROOT/pages/insight.adoc

```diff
@@ -25,45 +23,38 @@ Insight helps you answer questions about your integrated apps, such as:
 * What was the result of a transaction?
 * If something went wrong during processing, at what point did the failure occur?
 
-Using the Insight Dashboard, you can view events that occur within the flows that handle your business transactions. Further, you can drill down into event-related data to access information that can help you analyze the root cause of failures, isolate performance bottlenecks, and test for compliance with company procedures.
+Use the Insight Dashboard to view events that occur within the flows that handle your business transactions. You can drill down into event-related data to access information that can help you analyze the root cause of failures, isolate performance bottlenecks, and test for compliance with company procedures.
```

Comment:
Since it's a separate sentence now, maybe remove the You can & help you & access information? 
"To  analyze the root cause of failures, isolate..., and test ..., drill down into event-related data."